1.9/networking/service-and-pod-networking.md

```diff
@@ -0,0 +1,422 @@
+---
+post_title: Service and Pod Networking
+menu_order: 0.5
+---
+
+This document describes how to specify networking modes and assign ports in the service definition for a DC/OS Service you are creating.
+
```

Comment:
yes, DCOS users need to know about the marathon networking API changes. for example, they may have built external tooling that depends on the old (deprecated) APIs that are no longer emitted by Marathon